service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcTaskAggregateService.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.adaptors.gp2gp.ehr.EhrExtractStatusValidator;
+import uk.nhs.adaptors.gp2gp.ehr.SendEhrExtractCore;
+import uk.nhs.adaptors.gp2gp.ehr.model.EhrExtractStatus;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class GpcTaskAggregateService {
```

Comment:
Maybe we call this called `DetectTranslationCompleteService`? That is what it will do after we iterate on our first few translations stories.